test/unit/sql/select.test.js

```diff
@@ -232,4 +232,15 @@ suite(Support.getTestDialectTeaser('SQL'), function() {
     });
 
   });
+
+  suite('raw query', function () {
+    test('raw replacements', function () {
+      expectsql(sql.selectQuery('User', {
+        attributes: ['*'],
+        having: ['name IN (?)', [1, 'test', 3, "derp"]]
+      }), {
+        default: "SELECT * FROM [User] HAVING name IN (1,'test',3,'derp');"
```

Comment:
I meant bind parameters in the sense of prepared queries, but actually looking at the docs it seems that arrays are not supported

You could call sequelize.escape manually instead and you would get a postgres ARRAY[], I think this is kinda strange though and maybe not the best way to do it

In any case, this PR should bring back exactly the same behavior as in pre-3.14, which means breaking people's code if they relied on post-3.14 behavior for postgres